2019-01/PITCHME.md

```diff
@@ -176,5 +175,26 @@ Note:
 Still for the context, other projects we are using behind the scenes.
 
 ---
-@title[What's in it for SUSE?]
+@title[What's the SUSE state in the community]
 
+#### State of openSUSE in the OpenStack community
+
+@ul[](false)
+- OpenStack tests against openSUSE Leap using our packaging
+- When it breaks, they ping dirk on irc
+- Occassionally dirk is not available
+- We need to support the community and be fixing issues before they arise
```

Comment:
We need a better, more sustainable support for the community and take care of issues before they arise